params.go

```diff
@@ -169,6 +169,16 @@ var (
 		15,
 		"Period T between disasters in deterministic case and E[T] in stochastic case.",
 	)
+	disasterPeriodVisible = flag.Bool(
+		"disasterPeriodVisible",
+		true,
+		"Whether disasterPeriod is visible to agents",
+	)
+	disasterStochasticPeriodVisible = flag.Bool(
+		"disasterStochasticPeriodVisible",
+		true,
+		"Whether stochasticPeriod is visible to agents",
+	)
```

Comment:
I'd move this below the variable where it makes (in)visible. 
Also @JamesTev disasterPeriodStochastic or disasterStochasticPeriod?

Reply:
Not too fussed. `disasterStochasticPeriod` is how it was defined originally IIRC but the former kind of makes more sense. Up to Eirik.